cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1515,10 +1514,7 @@ ReconciliationState createReconciliationState(Reconciliation reconciliation, Kaf
         }
 
         Future<ReconciliationState> topicOperatorRoleBinding() {
-            String watchedNamespace = topicOperator != null ? topicOperator.getWatchedNamespace() : null;
             return withVoid(roleBindingOperator.reconcile(
-                    watchedNamespace != null && !watchedNamespace.isEmpty() ?
```

Comment:
Why do you think we don't need to distinguish between the `watchedNamespace` and `namespace` here (and in few other places) anymore? Do you deal with it somewhere else?